code/modules/mob/new_player/new_player.dm

```diff
@@ -490,6 +490,15 @@
 	. += ""
 	. += "Game Mode: [GLOB.master_mode]"
 
+	if(SSticker.current_state == GAME_STATE_FINISHED)
+		if(SSticker.mode.round_end_time == 0)
+			. += "Time To Restart: Ongoing Vote [SSticker.delay_end ? "(DELAYED)" : ""]"
+			return
+		if(SSticker.mode.round_end_time < 0)
+			. += "Time To Restart: SOON [SSticker.delay_end ? "(DELAYED)" : "(RESTART NEEDED)"]"
```

Comment:
```suggestion
			. += "Time To Restart: SOON [SSticker.delay_end ? "(DELAYED)" : "(MANUAL RESTART NEEDED)"]"
```
Assuming I interpreted this correctly, worth clarifying it needs to be done manually.